src/client/src/mockData/azureServiceOptions.ts

```diff
@@ -110,32 +119,39 @@ export const messages = defineMessages({
   }
 });
 
+export const servicesEnum = {
+  HOSTING: "Cloud Hosting",
```

Comment:
I think ideally this enum would be so that for each item in `serviceEnum` the message `<enum item>Service.title` exists. See the other comment where you actually use the message in AzureSubscriptions container for more context

Reply:
Do you mean something like this?
messages = {"Cloud Hosting Title": {id: whateverID, defaultMessage: "Host your web app in the cloud", [..]} }

or this: 
const servicesEnum = {
  HOSTING: {name: "Cloud Hosting", message: "Host your web app in the cloud"}
  [...]
}